### ISSUE TITLE
Teaching & Learning Resource Center MASTER ISSUE TRACKER

### ISSUE THREAD

**Issue by scottofletcher** [participedia/frontend#899]
TLRC submissions to be open to the public. Editing privileges are more advanced:

- Only admins and the original submitter can take down/remove uploaded/linked content (files, links, images, videos, audio)
- Other users can only edit the data fields and add content (files, links, images, videos, audio)
- All users can download/view content (files, links, images, videos, audio) NOTE: admins and original submitter will have to make sure other users are not downloading the content, changing it, and then re-uploading it (not that that would ever happen...)

**Comment by scottofletcher** [comment]
Need: design for landing page. 
- what should replace the map? 
- card view category distinctions (eg. homepage has 'CASE', 'METHOD', 'FEATURED ORGANIZATION' etc. Would it be possible to display the content type (dependent on the value in that data field)? See `[here ](https://drive.google.com/drive/folders/1eG-ngKBZN4Z9VTX96w4AYmtpnGdsdJcf)` for full list (under heading 'Resource Content & Design')

**Comment by scottofletcher** [comment]
Need: design for landing page.

- what should replace the map?
- card view category distinctions (eg. homepage has 'CASE', 'METHOD', 'FEATURED ORGANIZATION' etc. Would it be possible to display the content type (dependent on the value in that data field)? See [here](https://drive.google.com/drive/folders/1eG-ngKBZN4Z9VTX96w4AYmtpnGdsdJcf) for full list (under heading 'Resource Content & Design')

**Comment by scottofletcher** [comment]
Update: need at least one of the media sections (files, links, images, videos, audio) to be filled out before user can publish. This is the only entry type that requires an attachment/link

**Comment by jesicarson** [comment]
Hi @scottofletcher, I just noticed the public launch label here, but TLRC is currently scheduled as a post-launch update, following localization - if this needs to be shifted in priority order, please bring up in our next meeting with Pat.  
Also, about the required media thing - make sure this is noted in the fields spreadsheet for TLRC so its very clear for @ascott when she gets to it.

**Comment by plscully** [comment]
It was also my understanding that TLRC will not be part of P1, but will be part of P2

**Comment by scottofletcher** [comment]
Yes, sorry, I didn't change that label - it must have been labelled as such back when we first created the issue. 
I'll update the spreadsheet (which is here btw: https://drive.google.com/open?id=1xtRDs1rg8sglPIdmMnIgwyvl25g1Y-cyvv7ueryVhoo)
